finagle-backend/src/main/scala/sttp/client3/finagle/FinagleBackend.scala

```diff
@@ -228,7 +228,7 @@ class FinagleBackend(client: Option[Client] = None) extends SttpBackend[TFuture,
         Some(new SttpClientException.ConnectException(request, e))
       case e: com.twitter.finagle.ChannelClosedException => Some(new SttpClientException.ReadException(request, e))
       case e: com.twitter.finagle.IndividualRequestTimeoutException =>
-        Some(new SttpClientException.ReadException(request, e))
+        Some(new SttpClientException.TimeoutException(request, e))
```

Comment:
For sure it is the only backend that needs this change ?

Reply:
@Pask423 there are also individual adapters for Akka and Armeria in this PR. Other backends probably just emit standard java timeout exceptions.

Reply:
Ok  then, my bad

Reply:
Yes, the unchanged backends emit java native exceptions.